v2v-helper/migrate/migrate.go

```diff
@@ -259,16 +316,20 @@ func (migobj *Migrate) LiveReplicateDisks(vminfo vm.VMInfo) (vm.VMInfo, error) {
 
 func (migobj *Migrate) ConvertDisks(vminfo vm.VMInfo) error {
```

Comment:
For the sake of consistency, all function calls for operations on Volumes should probably be named accordingly. I think this should be: `ConvertVolumes`.

Reply:
Fixed